app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -21,6 +21,11 @@ keywords:
   - teaching
   - teacher
   - special school
+quote:
+  q-jane-wilkinson:
+    text: "I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit. And another school that offered support for children with a variety of learning needs. They were great at teaching me and letting me practice my classroom management skills early on. I was able to research and understand ways to differentiate resources for children who were on the Autistic Spectrum. And I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified."
+    name: "Jane Wilkinson"
```

Comment:
```suggestion
    name: "Jane Wilkinson, former teacher and teacher training adviser"
```
@garethwilliamsdesign Are you happy if we put this on one line, given how long the quote is already?

Reply:
!00%!!!